### PR TITLE
registry layout updatez

### PR DESCRIPTION
**Pull request by susanev**
this needs to ship at the same time as https://github.com/pulumi/pulumi-hugo/pull/2977

this work does not include removing api.html and setting the api docs pages to use package.html because that layout is set over in pulumi/pulumi so this first needs to ship, then i can make the change to update that to use package, then ill come back and remove api.html. lol sob forever.

after this lands i will...

- [ ] do the work to remove the need for package layout at all
- [ ] then update pulumi/pulumi to not add the api layout
- [ ] then do community prs to remove the layouts on index and install and update the terraform bridge sample project

to do

- [x] api docs nav seems to have disappeared 
- [x] on this page has extra space
- [x] figure out registry card
- [x] add layouts to package list
- [x] add left nav to packages list
- [x] add left nav to package pages
- [x] style left nav for package pages
- [x] h1 issues
- [x] add arrow icon to all packages link
- [x] add features packages to packages left nav
- [x] add accordion to registry card

### CODE REVIEW

File: themes/default/layouts/partials/registry/package/breadcrumb.html [hunk 1]
```diff
@@ -1,29 +0,0 @@
-<!-- Based on https://gohugo.io/content-management/sections/#example-breadcrumb-navigation -->
-
-<ol class="docs-breadcrumb">
-    {{ template "registrybreadcrumbnav" (dict "p1" . "p2" .) }}
-</ol>
-
-{{ define "registrybreadcrumbnav" }}
```
Comment (susanev) [inline themes/default/layouts/partials/registry/package/breadcrumb.html:7]: some of this code was dropped and needs to be added to the breadcrumb component over in hugo; as its accounting for things like the api docs issue

File: themes/default/layouts/partials/registry/package/right-nav.html [hunk 2]
```diff
@@ -1,64 +0,0 @@
-<div class="mt-2 hidden lg:block">
```
Comment (sean1588) [inline themes/default/layouts/partials/registry/package/right-nav.html:1]: good idea to have this managed in one place in hugo now :star2:

File: themes/default/layouts/registry/package.html [hunk 3]
```diff
@@ -0,0 +1,123 @@
+{{ define "main" }}
```
Comment (sean1588) [inline themes/default/layouts/registry/package.html:1]: thank you so much for consolidating this to one file now. this was much needed and i think will make this so much easier to work with going forward :star_struck:

**Review by sean1588 — APPROVED** [review]
amazing work here @susanev! thanks so much for all the things you did to improve our code-cleanliness!

File: themes/default/layouts/partials/registry/package-list.html [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+            A &ndash; Z Packages
+            <span class="all-count ml-4 py-0.5 px-3 rounded-full bg-violet-600 text-white text-lg">
+                {{ len $allPackages }}
+            </span>
+        </h2>
+        <p class="px-4 mt-0">
+            <a href="{{ relref . "/docs/using-pulumi/pulumi-packages/contribute-to-pulumi-registry" }}" class="font-display underline text-lg">Contribute your own packages</a>
+        </p>
+        <div class="flex flex-wrap">
+            {{/* Capture the current (i.e., Page) context so it can be passed into the partial, allowing the partial to build proper relrefs. */}}
```
[the remark this answers is not in the document]
Comment (susanev) [reply themes/default/layouts/partials/registry/package-list.html:81]: ah that was already there, but ill update this!!

File: themes/default/layouts/partials/registry/package/package-alert.html [hunk 5]
```diff
@@ -1,51 +1,94 @@
...
+        </div>
+    </div>
 {{ else if (eq $package "azure-native-v2") }}
-<div class="text-xs mt-4 note note-warning">
-    <i class="fas fa-info-circle text-blue-600 mr-1 my-1"></i>
-    <span>
-        Azure Native v2 is in preview. Use <a class="text-blue-600 underline" href="/registry/packages/azure-native/">Azure Native GA</a> for a fully supported version.
-    </span>
-</div>
+    <div class="note note-info">
```
[the remark this answers is not in the document]
Comment (susanev) [reply themes/default/layouts/partials/registry/package/package-alert.html:82]: no this is a mistake, ill fix it

**Pull request merged**